components/setting_picture.jsx

```diff
@@ -286,6 +269,55 @@ export default class SettingPicture extends Component {
             );
         }
 
+        let buttonRender;
+        if (this.props.onSubmit) {
+            buttonRender = (
+                <span>
+                    <input
+                        data-testid='uploadPicture'
+                        ref={this.selectInput}
+                        className='hidden'
+                        accept='.jpg,.png,.bmp'
+                        type='file'
+                        onChange={this.handleFileChange}
+                        disabled={this.props.loadingPicture}
+                        aria-hidden={true}
+                        tabIndex='-1'
+                    />
+                    <button
+                        data-testid='inputSettingPictureButton'
+                        className='btn btn-sm btn-primary btn-file sel-btn'
+                        disabled={this.props.loadingPicture}
+                        onClick={this.handleInputFile}
+                        aria-label={localizeMessage('setting_picture.select', 'Select')}
+                    >
+                        <FormattedMessage
+                            id='setting_picture.select'
+                            defaultMessage='Select'
+                        />
+                    </button>
+                    <button
+                        tabIndex={disableSaveButtonFocus ? '-1' : '0'}
+                        data-testid='saveSettingPicture'
+                        disabled={disableSaveButtonFocus}
+                        ref={this.confirmButton}
+                        className={confirmButtonClass}
+                        onClick={this.props.loadingPicture ? this.returnTrue : this.handleSave}
```

Comment:
I'm not a huge fan of how this is being done.  Could we do something like this possibly?
```suggestion
                        onClick={this.handleSave}
```
and then in the `handleSave()` function, we have something like this 
```Javascript
handleSave() {
     e.preventDefault()
     if (this.props.loadingPicture) {
            return;
     }
    // ... Rest of code
}
```

Reply:
I like it...done.